fly.toml

```diff
@@ -12,7 +12,7 @@ swap_size_mb = 2048
 
 [env]
   DATABASE_URI = '/data/nwc.db'
-  LDK_LOG_LEVEL = '2'
+  LDK_LOG_LEVEL = '3'
   LOG_LEVEL = '5'
```

Comment:
this should be 4? same as in .env ?

Reply:
![image](https://github.com/user-attachments/assets/f914cba8-b858-46c9-87f9-38fb9f2ff412)

Info level is the one we want by default, which is 3

Reply:
in `.env.example` `LOG_LEVEL` is set to 4. in fly it's 5.